src/sentry/static/sentry/app/views/settings/organization/members/organizationMembersView.jsx

```diff
@@ -65,6 +81,18 @@ class OrganizationMembersView extends AsyncView {
     return `${org.name} Members`;
   }
 
+  onSearch = e => {
```

Comment:
nit: can we rename this to be `handleSearch` - we'll be turning on this rule in the future: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md#rule-details